crates/fast_compute/src/layer.rs

```diff
@@ -223,13 +282,46 @@ impl<T: TowerFamily, P: PackedTop<T>> ComputeLayer<T::B128> for FastCpuLayer<T,
 	fn sum_composition_evals(
 		&self,
 		_exec: &mut Self::KernelExec,
-		_log_len: usize,
-		_inputs: &[FSlice<'_, T::B128, Self>],
-		_composition: &Self::ExprEval,
-		_batch_coeff: T::B128,
-		_accumulator: &mut T::B128,
+		inputs: &SlicesBatch<FSlice<'_, T::B128, Self>>,
+		composition: &Self::ExprEval,
+		batch_coeff: T::B128,
+		accumulator: &mut Self::KernelValue,
 	) -> Result<(), Error> {
-		unimplemented!()
+		const BATCH_SIZE: usize = 64;
```

Comment:
Please leave a comment explaining this choice. Even if it's arbitrary, that should go in the comment.